tests/utils/metrics/test_gauge.py

```diff
@@ -29,13 +29,16 @@ def test_gauge_concurrent() -> None:
     tags = {"tag": "value"}
     gauge = Gauge(backend, name, tags)
 
+    workers = 4
+    barrier = Barrier(workers)
```

Comment:
This is great, I didn't know about it until today: https://docs.python.org/3/library/threading.html#barrier-objects